### DISCUSSION TITLE
Event is over, but still in the pulldown.

### DISCUSSION THREAD

**Discussion by HansF** [pretix/pretix#2779] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1503/188139249-fd95c966-33e8-45db-ab9f-75ceda772af2.png)

I don't have this issue with the other events, basically the event is over, as you can see in the screenshot above. 

![image](https://user-images.githubusercontent.com/1503/188139566-2bdd80b8-6b39-42df-9ea1-e904696233b3.png)

Nothing peculiar about the timeline settings. 
I do however still see this event in the pull down on the left. 
![image](https://user-images.githubusercontent.com/1503/188139867-454630bd-6ad3-47ae-b028-b322ecc3b2f6.png)

Not a big deal, but am I missing something?

**Comment by raphaelm** [comment, marked as answer]
I'm not sure what the problem is, the "pull down" is a list of events we think you might be looking for which can be filtered through search input. It does not matter if the event is over.

**Comment by HansF** [comment]
Hey Raphael,

I just realized that I might have misinterpreted the "pull down"  in the past. It seems like it was only for active events, not for all of them.

I thank you for your time.

Best,

Hans
